ingestion/src/metadata/ingestion/source/dashboard/sigma/models.py

```diff
@@ -0,0 +1,95 @@
+#  Copyright 2023 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""
+PowerBI Models
+"""
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+
+
+class AuthToken(BaseModel):
+    access_token: str
+    refresh_token: str
+    token_type: str
+
+
+class Workbook(BaseModel):
+    workbookId: str
+    name: Optional[str] = ""
```

Comment:
should we default to `None` instead of empty strings?

Reply:
yes will default to None here